src/DssSpell.t.sol

```diff
@@ -118,10 +140,14 @@ contract DssSpellTest is DssSpellTestBase {
         }
 
         vote(address(spell));
-        scheduleWaitAndCast(address(spell));
+        spell.schedule();
+        hevm.warp(now + pause.delay());
```

Comment:
This should use the `spell.nextCastTime()` as we may be outside office hours / weekend.

Reply:
Resolving as I think this must have been fixed already.